.github/workflows/main.yml

```diff
@@ -158,9 +145,9 @@ jobs:
       - name: Maven Install
         run: |
           export MAVEN_OPTS="${MAVEN_INSTALL_OPTS}"
-          ./mvnw install ${MAVEN_FAST_INSTALL} -am -pl '${{ matrix.modules }}'
+          ./mvnw install ${MAVEN_FAST_INSTALL} -am -pl $(echo '${{ matrix.modules }}' | cut -d' ' -f1)
```

Comment:
you can retain apostrophes: `'$( ... )'` here

Reply:
Yes, but that would be inconsistent with the one below, and seems strange since we are already splitting on space.